### PR TITLE
Updating with Butterfly Devnet info

### PR DESCRIPTION
**Pull request by eshon**

### CODE REVIEW

**Comment by eshon** [comment]
> Thanks!
> 
> I can't get https://nerpa.fildev.network/ to load (linked on lines 36 and 48). Am I doing this wrong? If this link is broken, let's remove from both locations.

@mishmosh - fixed the link, it was just missing the www

http://www.nerpa.fildev.network/

**Comment by mishmosh** [comment]
works now!

File: docs/build/start-building/interacting-with-the-network.md [hunk 1]
```diff
@@ -26,23 +26,40 @@ Here are some scripts to set up your own remote networks for development:
 - [Lotus Docker Image](https://github.com/openworklabs/filecoin-docker) for a simple Lotus node Docker container
 - [Filecoin Chart](https://github.com/openworklabs/filecoin-chart) for more complex Lotus node architectures managed with Kubernetes
 
-## Devnet
+## Devnets
 
-In order to make testing easier, a live devnet (developer network) is now available for use. We call this network the Nerpa Devnet (the 'nerpa' is a species of seal, and one of the smallest true seals). The Nerpa Devnet features 512MB sectors (in addition to 32GB and 64GB sectors) and reduced proofs parameters, so sealing is much faster than in the full Testnet version.
+#### Nerpa Devnet
```
Comment (har00ga) [inline docs/build/start-building/interacting-with-the-network.md:31]: Could these be moved up to `###` level? They feel a bit on the small side to me, think they may get lost in the surrounding text

File: docs/build/start-building/interacting-with-the-network.md [hunk 2]
```diff
@@ -26,23 +26,40 @@ Here are some scripts to set up your own remote networks for development:
 - [Lotus Docker Image](https://github.com/openworklabs/filecoin-docker) for a simple Lotus node Docker container
 - [Filecoin Chart](https://github.com/openworklabs/filecoin-chart) for more complex Lotus node architectures managed with Kubernetes
 
-## Devnet
+## Devnets
```
Comment (har00ga) [inline docs/build/start-building/interacting-with-the-network.md:30]: A one-liner here for context would be +1, perhaps something like "There are currently two developer networks available for users to try out: the 'Nerpa' devnet, which is recommended for developer audiences, and the Butterfly devnet, which is best suited for miners that are interested in testing out new Lotus features."

File: docs/build/start-building/interacting-with-the-network.md [hunk 3]
```diff
@@ -26,23 +26,40 @@ Here are some scripts to set up your own remote networks for development:
...
 
-In order to make testing easier, a live devnet (developer network) is now available for use. We call this network the Nerpa Devnet (the 'nerpa' is a species of seal, and one of the smallest true seals). The Nerpa Devnet features 512MB sectors (in addition to 32GB and 64GB sectors) and reduced proofs parameters, so sealing is much faster than in the full Testnet version.
+#### Nerpa Devnet
+*best for Developers*
+
+To make development and testing easier, a live, long-lived devnet (developer network) is available for use. We call this network the Nerpa Devnet (the 'nerpa' is a species of seal, and one of the smallest true seals). The Nerpa Devnet features 512MB sectors (in addition to 32GB and 64GB sectors) and reduced proofs parameters, so sealing is much faster than in the full Testnet version.
```
Comment (har00ga) [inline docs/build/start-building/interacting-with-the-network.md:34]: First sentence about development and testing would be good to move to the context line under the Devnets header, should you agree with that route. 🙂

**Pull request merged**